include/circt/Transforms/InsertMergeBlocks.h

```diff
@@ -0,0 +1,29 @@
+//===- InsertMergeBlocks.h - Insert Merge Blocks ----------------*- C++ -*-===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef CIRCT_TRANSFORMS_INSERTMERGEBLOCKS_H
+#define CIRCT_TRANSFORMS_INSERTMERGEBLOCKS_H
+
+#include "mlir/IR/Region.h"
+#include "mlir/Transforms/DialectConversion.h"
+
+namespace mlir {
```

Comment:
Somehow the transformation live in `mlir::circt` which shouldn't be the case. I'll fix the other transformations before I change this here.

Reply:
Fixed in https://github.com/llvm/circt/pull/3464.